packages/vue/src/components/__tests__/__snapshots__/setup-totp.spec.ts.snap

```diff
@@ -216,14 +217,9 @@ exports[`SetupTotp renders qrcode image as expected after onMounted is done 1`]
             class="amplify-flex amplify-authenticator__column"
           >
             
-            <img
```

Comment:
Seems odd that this would go away in the snapshot, do we expect this?

Reply:
Should be good now. I think there was an update to the testing framework or something because I didn't make changes here but the tests weren't waiting for the qr code to render properly